deliverables/README.md

```diff
@@ -11,17 +13,41 @@ Sign Up for Cloud9 and create a basic Rails project.
 ###Intro to Ruby
 ####Required
 
-What is an "interpreted" language?<br>
-What is IRB?<br>
-What are Objects?<br>
-What are Methods?<br>
-What are Classes?<br>
-What are Blocks?<br>
-What is an Array?<br>
-What is an Iterator?<br>
-What are hashes?<br>
-What is a library?<br>
-What is a gem?
+What is an "interpreted" language?<br>  
+a language where the code is not in the native language of the machine but is interpreted by an installed program  
```

Comment:
I don't really have a sense of what that means, and I can't find anywhere that approaches the topic from a beginner's perspective.